In [47]:
import pandas as pd

df = pd.read_csv("vanguard_master_cleaned.csv")

clients = df.drop_duplicates(subset="client_id").copy()

In [48]:
# Check The Unique Client Dataset

clients.shape

(50488, 16)

In [49]:
# Average age
round(clients["clnt_age"].mean(), 2)

np.float64(47.32)

In [50]:
# Average Tenure

round(clients["clnt_tenure_yr"].mean(), 2)

np.float64(12.03)

In [51]:
# Gender Split

clients["gendr"].value_counts()
round(clients["gendr"].value_counts(normalize=True) * 100, 2)

gendr
U    34.23
M    33.57
F    32.20
X     0.00
Name: proportion, dtype: float64

In [52]:
# Are they new or long-standing investors?

def tenure_group(years):
    if years < 5:
        return "New client"
    elif years <= 10:
        return "Established client"
    else:
        return "Long-standing client"

clients["tenure_group"] = clients["clnt_tenure_yr"].apply(tenure_group)

In [53]:
# Group counts and percentage

clients["tenure_group"].value_counts()

tenure_group
Long-standing client    25783
Established client      20926
New client               3779
Name: count, dtype: int64

In [54]:
# Percentage of the group

round(clients["tenure_group"].value_counts(normalize=True) * 100, 2)

tenure_group
Long-standing client    51.07
Established client      41.45
New client               7.48
Name: proportion, dtype: float64

In [55]:
# Check high balance clients

clients["is_high_balance"].value_counts()

is_high_balance
False    38884
True     11604
Name: count, dtype: int64

In [56]:
# Percentage of high balance clients
round(clients["is_high_balance"].value_counts(normalize=True) * 100, 2)

is_high_balance
False    77.02
True     22.98
Name: proportion, dtype: float64

In [57]:
# Check multi-account clients

clients["is_multi_account"].value_counts()

is_multi_account
False    39722
True     10766
Name: count, dtype: int64

In [58]:
# Check percentages of multi-account clients

round(clients["is_multi_account"].value_counts(normalize=True) * 100, 2)

is_multi_account
False    78.68
True     21.32
Name: proportion, dtype: float64

## Phase 1 Findings: Demographic Analysis

After removing duplicate client records, the dataset contains **50,488 unique clients**. This client-level dataset was used for demographic analysis to avoid counting the same client multiple times across different process steps.

The average client age is **47.32 years**, which suggests that the primary users of the Vanguard digital process are middle-aged investors. The gender distribution is fairly balanced: **34.23% unknown/unspecified**, **33.57% male**, and **32.20% female**.

The average client tenure is **12.03 years**, showing that most users are not new investors. Based on tenure groups, **51.07%** of clients are long-standing clients, **41.45%** are established clients, and only **7.48%** are new clients. This suggests that the primary clients using the process are mostly long-standing or established Vanguard investors.

For client segments, **22.98%** of unique clients are classified as high-balance clients, while **21.32%** are multi-account clients. These groups are important for the next phases because they may behave differently during the online process and may need closer analysis for drop-off points or personalized support.

Overall, the primary clients using this process appear to be middle-aged, relatively long-tenured investors, with a balanced gender distribution and a meaningful segment of high-balance and multi-account clients.